policybot/pkg/storage/spanner/query.go

```diff
@@ -68,6 +68,74 @@ func (s store) QueryIssuesByRepo(context context.Context, orgLogin string, repoN
 	return err
 }
 
+func (s store) QueryTestResultByName(context context.Context, testName string, cb func(*storage.TestResult) error) error {
```

Comment:
This probably should take an org & repo and constrain the query. Otherwise, it'll return  any tests by the given name across any org &  repo, which is likely not what the upper layer of the code wants.